ispcrt/CMakeLists.txt

```diff
@@ -133,16 +133,23 @@ foreach(PRNAME ${LIB_LIST})
 
     # Security options
     if (MSVC)
-      # Stack canaries
-      target_compile_options(${PRNAME} PRIVATE /GS)
-      # Control flow guard
-      target_link_options(${PRNAME} PRIVATE /GUARD:CF)
-    elseif (APPLE)
+        # Stack canaries
+        target_compile_options(${PRNAME} PRIVATE /GS)
+        # Control flow guard
+        target_link_options(${PRNAME} PRIVATE /GUARD:CF)
     else()
-      # Enable stack protector
-      # NOT to assume that null pointer deference does not exist
-      # Assume that signed overflow always wraps
-      target_compile_options(${PRNAME} PRIVATE -fstack-protector-strong -fno-delete-null-pointer-checks -fwrapv)
+        # Enable stack protector
+        # NOT to assume that null pointer deference does not exist
+        # Assume that signed overflow always wraps
+        target_compile_options(${PRNAME} PRIVATE -fstack-protector-strong -fno-delete-null-pointer-checks -fwrapv)
+        target_compile_options(${PRNAME} PRIVATE -fPIC)
```

Comment:
I think we already have the libraries built with position independent code:
```
set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```
https://github.com/ispc/ispc/pull/2594/files#diff-656ce737b6f5d8ae5ed47377acf495bbdc41d00cbb6df559df28fec0fd71d15cR20